recipe/nsis_activate.bat

```diff
@@ -0,0 +1 @@
+@SET "PATH=%CONDA_PREFIX%\NSIS;%PATH%"
```

Comment:
We should also have a deactivate script. This may require storing a backup of the path during activate.

Reply:
Also we normally just call them `activate.bat` and `deactivate.bat` as the `nsis` is already clear by being part of this recipe.

Reply:
> We should also have a deactivate script.

Yes! That was a brain fart on my end here.

> just call them `activate.bat` and `deactivate.bat`

For the recipe itself I think this is a good call. But then I should change the filename when copying in `bld.bat`, right? Skimming the code for `conda`/`conda-build` I didn't see any indication those scripts will get renamed by them. So if not named acc. to the package, they might clobber de-/activate scripts from other packages.

Reply:
Correct. Pretty much most activate/deactivate script installs mimic the [`toolchain` recipe]( https://github.com/conda-forge/toolchain-feedstock ). Here's the [Windows build script for the `toolchain`]( https://github.com/conda-forge/toolchain-feedstock/blob/f87689631ac261cb705b9ade2553edf439a56d9f/recipe/bld.bat ). One could probably be more clever and substitute `toolchain` in that recipe with `%PKG_NAME%`.